tests/utils.py

```diff
@@ -0,0 +1,37 @@
+# Query count helpers, copied from django source
```

Comment:
i've added this utils file since this package supports django <3.1, and `assertNumQueries` was only introduced therein